Validation/MuonHits/plugins/BuildFile.xml

```diff
@@ -0,0 +1,13 @@
+<library   file="*.cc" name="ValidationMuonHitsPlugin">
+<use   name="Geometry/CSCGeometry"/>
+<use   name="Geometry/DTGeometry"/>
+<use   name="Geometry/RPCGeometry"/>
+<use   name="Geometry/GEMGeometry"/>
+<use   name="Geometry/Records"/>
+<use   name="SimDataFormats/TrackingHit"/>
+<use   name="SimDataFormats/Vertex"/>
+<use   name="Validation/MuonHits"/>
+<use   name="root"/>
+<use   name="clhep"/>
```

Comment:
@dildick where is it needed?

Reply:
In MuonSimHitsValidAnalyzer.h you have two CLHEP headers that look unused, am I correct?